File: backend/src/main/java/com/edoc/backend/dto/AuthRequest.java
package com.edoc.backend.dto;

import com.edoc.backend.enums.Role;
import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@ToString
public class AuthRequest {
  Long id;
  String username;
  String password;
  Role role;
}

File: frontend/src/model/AuthRequest.js

export default class AuthRequest {
    constructor(
        id = null,
        username = null,
        password = null,
        role = null, token = null
    ) {
        this.id = id;
        this.username = username;
        this.password = password;
        this.role = role;
        this.token = token
    }
}

